routes/apply_discount.js

```diff
@@ -232,6 +234,59 @@ const responseSchema = {
   'type': 'object',
   'additionalProperties': false,
   'properties': {
+    'buy_together': {
+      'type': 'array',
+      'maxItems': 300,
+      'items': {
+        'type': 'object',
+        'required': [ 'discount' ],
+        'additionalProperties': false,
+        'properties': {
+          'product': {
+            'type': 'object',
+            'maxProperties': 100,
+            'patternProperties': {
+              '^([a-f0-9]{24})$': {
+                'type': 'object',
+                'properties': {
+                  'quantity': {
+                    'type': 'number',
+                    'minimum': 0,
+                    'maximum': 9999999,
+                    'description': 'Product quantity'
+                  },
+                },
+                'required': ['quantity']
```

Comment:
```suggestion
                'type': 'number',
                'minimum': 1,
                'maximum': 9999999,
                'description': 'Product quantity to buy'
```

Reply:
E o mínimo aqui tem que ser 1 né, não podemos retornar um "compre junto com 0 bananas e ganhe 10% off" 😄

Reply:
Pensei isso, porque o app permite não ter quantidade minima e ai nessas situações, poderia renderizar um compre junto com 3 produtos relacionados, se for a condição mínima do cupom com o % configurado lá, mas pra não complicar faz sentido